dist/index.html

```diff
@@ -6,10 +6,95 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
+    <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="container-parent">
+      <div class="container-header">
+        <div class="container-logo">
+            <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" border="0" id="mainlogo">
+        </div>
+        <div class="container-navSection">
+          <div class="container-nav">
+                <div class="menu"> <a href="home.asp" class="fas fa-home">Home</a> </div>
+                <div class="menu"> <a href="projects.asp" class="fas fa-desktop">Projects</a> </div>
+                <div class="menu"> <a href="about.asp" class="fas fa-user">About_Me</a> </div>
+                <div class="menu"> <a href="contact.asp" class="fas fa-envelope">Contact</a> </div>
+          </div>
+          <div class="container-moto">
+            <h2>Custom Designs · Marketing · Branding · And More </h2>
+          </div>
+        </div>
+      </div>
+      <div class="container-body">
+        <div class="content">
+          <div class="content-left">
+            <span class ="ownerContact" style="color: #336D61; font-size: 3em; font-family: AlexandriaFLF;">
+              CONTACT
+            </span>
+            </h2>
+              <hr>
+            <p style="text-align: center;">
+              <span class="ownerName">
+                LYANDRA SICA<br/>
+              </span>
+              <span class="ownerDetails">
+                Creative Designer<br/><br/>
+                413 Myrtle Avenue <br/>
+                West Islip, NY 11795 <br/>
+                creative@seekadesign.com <br/>
+                631.605.5782 <br/>
+              </span>
+                <hr>
+              <span class="share">
+                FOLLOW. LIKE. SHARE. <br/>
+                <div class="social">
+                  <div class="circle-icon-green">
+                    <a href="https://www.facebook.com/SeekADesign/" target="_blank" class="fab fa-facebook-f"></a> <br/>
+                  </div>
+                  <div class="circle-icon-purple">
+                    <a href="https://www.facebook.com/SeekADesign/" target="_blank" class="fab fa-twitter"></a> <br/>
+                  </div>
+                  <div class="circle-icon-green">
+                    <a href="https://www.instagram.com/seekadesign/" target="_blank" class="fab fa-instagram"></a> <br/>
+                  </div>
+                </div>
+              </span>
+              <span class="at">
+                @SeekADesign
+              </span>
+            </p>
+          </div>
+          <div class="content-right">
+            <h2>Questions?<br/>
```

Comment:
**Lines 69-70:**

Can you change this to two separate `<h2>` tags, with no `<br/>` tag?